omod/src/main/webapp/pages/htmlform/flowsheet.gsp

```diff
@@ -24,6 +24,13 @@
     flowsheet.setDefaultLocationId(${ defaultLocationId });
     flowsheet.setRequireEncounter(${ requireEncounter });
 
+    var fwEncounterTypes = {};
```

Comment:
What does fw stand for? I'd just be explicit and do flowsheetEncounterTypes if fw=flowshseet